bindings/redis/redis.go

```diff
@@ -66,21 +66,50 @@ func (r *Redis) Ping() error {
 }
 
 func (r *Redis) Operations() []bindings.OperationKind {
-	return []bindings.OperationKind{bindings.CreateOperation}
+	return []bindings.OperationKind{
+		bindings.CreateOperation,
+		bindings.DeleteOperation,
+		bindings.GetOperation,
+	}
 }
 
 func (r *Redis) Invoke(ctx context.Context, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error) {
-	if val, ok := req.Metadata["key"]; ok && val != "" {
-		key := val
-		_, err := r.client.Do(ctx, "SET", key, req.Data).Result()
-		if err != nil {
-			return nil, err
+	if key, ok := req.Metadata["key"]; ok && key != "" {
+		switch req.Operation {
+		case bindings.DeleteOperation:
+			var arg []string
+			arg = append(arg, req.Metadata["key"]) //at least one key
+			index := 1
```

Comment:
Why is `index` starting from 1, if you always do `index-1` below? Shouldn't it start from 0?

Reply:
For consistency, the default [key] must exist. At the same time, in order to support multiple keys, the latter will start from key1, which means that the index also starts from 1. If it is changed to a single key, this problem does not exist